### PR TITLE
Scope2

### PR DESCRIPTION
**Pull request by oryrosin**

### CODE REVIEW

**Review by alonisser — DISMISSED** [review]

File: src/contrast/pages/Home/HomePage.jsx [hunk 1]
```diff
@@ -111,7 +121,15 @@ export default function HomePage() {
               <Button
                 extraClass="text-lg sm:text-sm "
                 black
-                onClick={copyToClipboard}>
+                onClick={() => {
+                  console.log("first");
+                  ReactGA.event({
+                    category: "HomePage",
+                    action: "cite",
```
[the remark this answers is not in the document]
Comment (oryrosin) [reply src/contrast/pages/Home/HomePage.jsx:128]: i m trying things. it will be removed later

File: index-contrast.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <script
       async
       src="https://www.googletagmanager.com/gtag/js?id=G-9XP6GJSL3K"></script>
```
Comment (alonisser) [inline index-contrast.html:7]: you sure you need both this and the library initialization? quite sure that's not true

File: src/contrast/AppContrast.jsx [hunk 3]
```diff
@@ -6,6 +6,14 @@ import { ToastContainer, toast } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
 import ContrastScreens from "./ContrastScreens.jsx";
 import usePreventNumberInputScroll from "../Utils/PreventScrollEffect.ts";
+import ReactGA from "react-ga4";
+ReactGA.initialize("G-9XP6GJSL3K", {
```
Comment (alonisser) [inline src/contrast/AppContrast.jsx:10]: what about uncontrust?

**Review by alonisser — APPROVED** [review]
Approved with comment

File: index-contrast.html (hunk 2 above)
Comment (oryrosin) [reply index-contrast.html:7]: the react-ga package, apparently doesnt work

File: src/contrast/AppContrast.jsx (hunk 3 above)
Comment (oryrosin) [reply src/contrast/AppContrast.jsx:10]: כשאהיה סגור על משהו שעובד- אטפל גם בו

**Pull request merged**